### PR TITLE
Update cgroups.md

### PR DESCRIPTION
**Pull request by wenjianhn**
Add an example of uber-go/automaxprocs which is used by lots of applications to avoid CPU throttling.

Related to https://github.com/uber-go/automaxprocs/issues/49

### CODE REVIEW

File: content/en/docs/concepts/architecture/cgroups.md [hunk 1]
```diff
@@ -103,6 +103,7 @@ updated to newer versions that support cgroup v2. For example:
 * If you run [cAdvisor](https://github.com/google/cadvisor) as a stand-alone
  DaemonSet for monitoring pods and containers, update it to v0.43.0 or later.
 * If you use JDK, prefer to use JDK 11.0.16 and later or JDK 15 and later, which [fully support cgroup v2](https://bugs.openjdk.org/browse/JDK-8230305).
+* If you use uber-go/automaxprocs, update it to v1.5.1 or later.
```
Comment (tengqm) [inline content/en/docs/concepts/architecture/cgroups.md:106]: is this package relevant to cgroup v2?

Comment (shannonxtreme) [reply content/en/docs/concepts/architecture/cgroups.md:106]: @bobbypage do you have an insight here?

Comment (wenjianhn) [reply content/en/docs/concepts/architecture/cgroups.md:106]: @tengqm like JDK, automaxprocs depends on cgroupfs to support container awareness.

Without it, an application is likely to hit the [cgroup throttling issue](https://danluu.com/cgroup-throttling) issue. I think it's a block issue of migrating to v2. AFAIK, it would lots of companies that depend on Go.

Comment (tengqm) [reply content/en/docs/concepts/architecture/cgroups.md:106]: @wenjianhn Thanks for the clarification.

```suggestion
* If you are using [uber-go/automaxprocs](https://github.com/uber-go/automaxprocs) package, make sure
  the version you use is v1.5.1 or higher.
```

Comment (wenjianhn) [reply content/en/docs/concepts/architecture/cgroups.md]: @tengqm done

**Comment by tengqm** [comment]
Thanks.
/approve
